contracts/resolver/IZNSStringResolver.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.18;
+
+
+interface IZNSStringResolver {
+    /**
+     * @param newString The new domain owner
```

Comment:
change the order of these.
also, `newString` comment is not correct. it's not the owner, it's the string that owner assigns which a domain will resolve to